cross/java-11-openjdk/Makefile

```diff
@@ -86,8 +86,8 @@ include ../../mk/spksrc.cross-cc.mk
 
 # Filter out any -jN since java's configure will autodetect number of CPU's
 # Use make JOBS=N to set number of CPU's in this package
-TMP_MAKEFLAGS:= $(MAKEFLAGS)
-MAKEFLAGS = $(filter-out -j%, $(TMP_MAKEFLAGS))
+TMP_MAKEFLAGS:= $(COMPILE_MAKE_OPTIONS)
+COMPILE_MAKE_OPTIONS = $(filter-out -j%, $(TMP_MAKEFLAGS))
```

Comment:
You will not necessarily address all cases with this as `MAKEFLAGS` follows various `MAKELEVELS` and may still be in effect containing `-jX`.  As such in theory you could still get `make $(MAKEFLAGS) $(COMPILE_MAKE_OPTIONS)` and resulting in `make -jX JOBS=X` with a build failure.

Adding an exceptional `COMPILE_TARGET` such as `-@MAKEFLAGS= $(MAKE) $(COMPILE_MAKE_OPTIONS)` circumvent this entirely.